pinot-broker/src/main/java/com/linkedin/pinot/broker/requesthandler/BrokerRequestHandler.java

```diff
@@ -29,5 +30,6 @@
 
   void shutDown();
 
-  BrokerResponse handleRequest(JSONObject request, @Nullable RequesterIdentity requesterIdentity) throws Exception;
+  BrokerResponse handleRequest(JSONObject request, @Nullable RequesterIdentity requesterIdentity,
```

Comment:
Did you want to deprecate the earlier one and add a new one instead?

Reply:
yes, done